src/common/is-service-worker-supported.ts

```diff
@@ -0,0 +1,30 @@
+import semver from 'semver'
+import { browserName, browserVersion } from 'react-device-detect'
+
+const minimum_required_browsers_versions: Record<string, string> = {
```

Comment:
@shayan-deriv, should we move this to the `constants` file? 🤔

Reply:
@jim-deriv not necessarily :thinking: because we are just using it in one place; that's why I wanted to keep everything in one place but if you think it's better to move it there , I will do so

Reply:
Alright @shayan-deriv, It's not that important LGTM